imports/plugins/included/taxes-avalara/client/containers/avalaraSettingsFormContainer.js

```diff
@@ -0,0 +1,153 @@
+import _ from "lodash";
+import { compose, withProps } from "recompose";
+import { composeWithTracker, registerComponent } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { Reaction, i18next } from "/client/api";
+import { AvalaraPackageConfig } from "../../lib/collections/schemas";
+import { AvalaraSettingsForm } from "../components";
+import { Logs as LogSchema } from "/lib/collections/schemas/logs";
+import { Countries } from "/client/collections";
+
+/**
+ * @file This is a container for AvalaraSettingsForm.
+ * @module avalaraSettingsFormContainer
+ */
+
+const handlers = {
+  /**
+    * handleSubmit
+    * @method
+    * @summary event handler for when new Avalara settings are submitted.
+    * @param {Object} event - event info.
+    * @param {Object} changedInfo - info about the new Avalara settings.
+    * @param {String} targetField - where to save the new settings in the Avalara Package.
+    * @return {null} - returns nothing
+    */
+  handleSubmit(event, changedInfo, targetField) { // eslint-disable-line no-unused-vars
```

Comment:
I feel like this UI stuff should be moved out of the container and into the component if possible. Generally I would like containers to contain only data.